src/Illuminate/Collections/Arr.php

```diff
@@ -597,6 +597,20 @@ public static function random($array, $number = null, $preserveKeys = false)
         return $results;
     }
 
+    /**
+     * Select and "pick" columns from a bi-dimensional array.
+     *
+     * @param  array  $array
+     * @param  array  $columns
```

Comment:
Shouldn't this be `@param  array|string  $columns` then, if we accept strings as well?